src/test/scala/org/greenrd/seraphim/BaseCommitTest.scala

```diff
@@ -0,0 +1,55 @@
+package org.greenrd.seraphim
+
+import java.io.{File, PrintWriter}
+
+import org.apache.commons.io.FileUtils
+import org.scalatest.FunSpec
+
+class BaseCommitTest extends FunSpec {
+
+  private val gitRepoName = "temp-git"
+
+  private def prepareGitRepo(): GitRepo = {
+    val gitRepoFolder = new File("target", s"$gitRepoName")
```

Comment:
This doesn't need to be done as a string substitution - can just do `gitRepoName` directly.